<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InBandOptionsClass</class>
 <widget class="QWidget" name="InBandOptionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>98</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbSettings">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblBlockSize">
        <property name="text">
         <string>Size of data in packet:</string>
        </property>
        <property name="buddy">
         <cstring>spbBlockSize</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spbBlockSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
           <string> Bytes</string>
          </property>
          <property name="minimum">
           <number>128</number>
          </property>
          <property name="maximum">
           <number>10240</number>
          </property>
          <property name="singleStep">
           <number>32</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblMaxBlockSize">
        <property name="text">
         <string>Maximum size of data in packet:</string>
        </property>
        <property name="buddy">
         <cstring>spbMaxBlockSize</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spbMaxBlockSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
           <string> Bytes</string>
          </property>
          <property name="minimum">
           <number>4096</number>
          </property>
          <property name="maximum">
           <number>102400</number>
          </property>
          <property name="singleStep">
           <number>32</number>
          </property>
          <property name="value">
           <number>10240</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblStanzaType">
        <property name="text">
         <string>Packet type:</string>
        </property>
        <property name="buddy">
         <cstring>cmbStanzaType</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cmbStanzaType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
